decidim-design/app/views/decidim/design/foundations/accessibility.html.erb

```diff
@@ -1,99 +1,99 @@
 <% content_for :heading do %>
-  Accessibility
+  <%= t(".accessibility_title") %>
 <% end %>
 
 <% content_for :description do %>
-  <p>Decidim follows the <u>Web Content Accessibility Guidelines (WCAG) 2.1</u>.</p>
+  <p><%= t(".decidim_follows") %><u><%= t(".web_content") %></u>.</p>
 <% end %>
 
 <% content_for :toc do %>
-  <a href="#heading-order">Illogical heading order</a>
-  <a href="#unique-h1">Unique H1</a>
-  <a href="#one-heading">Heading on important sections</a>
-  <a href="#color-contrast">Color contrast</a>
-  <a href="#links-buttons">Links and buttons</a>
-  <a href="#a11y-labels">Accessibility labels</a>
-  <a href="#adjacent-links">Adjacent links</a>
-  <a href="#aria">Use ARIA</a>
-  <a href="#elements-hidden">Elements hidden</a>
+  <a href="#heading-order"><%= t(".illogical_heading_order") %></a>
+  <a href="#unique-h1"><%= t(".unique_h1") %></a>
+  <a href="#one-heading"><%= t(".heading_on") %></a>
+  <a href="#color-contrast"><%= t(".color_contrast") %></a>
+  <a href="#links-buttons"><%= t(".links_and_buttons") %></a>
+  <a href="#a11y-labels"><%= t(".accessibility_labels") %></a>
+  <a href="#adjacent-links"><%= t(".adjacent_links") %></a>
+  <a href="#aria"><%= t(".use_aria") %></a>
+  <a href="#elements-hidden"><%= t(".elements_hidden") %></a>
 <% end %>
 
 <section id="heading-order" class="design__section">
-  <h2>Illogical heading order</h2>
+  <h2><%= t(".illogical_heading_order") %></h2>
 
-  <p>Every page should have a logical heading order when using the &#60;h1&#62;, &#60;h2&#62;, &#60;h3&#62;, &#60;h4&#62;, &#60;h5&#62; and &#60;h6&#62; heading elements.</p>
+  <p><%= t(".illogical_heading_paragraph") %></p>
 
-  <p>See further info <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_illogical_heading_order" target="_blank" rel="noopener noreferrer">here</a></p>
+  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_illogical_heading_order" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
```

Comment:
> @alecslupu I just committed this example and it didn't work (even when adding the HTML value to `t(".here_link")`.
> 
> Could you please try this out locally to make sure this work around works? I just want to make sure we're not diverting too much away from our original layout of the links per section with a **here** link on the front end.

Reply:
@greenwoodt , check the following PR. It should fix the issue. 

Also, check the external link pop-up, it's slightly off. 

```patch 
diff --git a/decidim-design/app/views/decidim/design/foundations/accessibility.html.erb b/decidim-design/app/views/decidim/design/foundations/accessibility.html.erb
index d2b6f3b13f..abf93bb57a 100644
--- a/decidim-design/app/views/decidim/design/foundations/accessibility.html.erb
+++ b/decidim-design/app/views/decidim/design/foundations/accessibility.html.erb
@@ -23,7 +23,7 @@
 
   <p><%= t(".illogical_heading_paragraph") %></p>
 
-  <p><%= t(".see_further_info", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_illogical_heading_order", target: "_blank", rel: "noopener noreferrer")) %></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_illogical_heading_order", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="unique-h1" class="design__section">
@@ -31,7 +31,7 @@
 
   <p><%= t(".unique_h1_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_unique_h1_on_every_page" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_unique_h1_on_every_page", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="one-heading" class="design__section">
@@ -39,7 +39,7 @@
 
   <p><%= t(".important_sections_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_heading_on_important_sections" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_heading_on_important_sections", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="color-contrast" class="design__section">
@@ -47,7 +47,7 @@
 
   <p><%= t(".color_contrast_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_color_contrast " target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_color_contrast", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="links-buttons" class="design__section">
@@ -55,7 +55,7 @@
 
   <p><%= t(".links_and_buttons_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_links_and_buttons" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_links_and_buttons", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="a11y-labels" class="design__section">
@@ -63,7 +63,7 @@
 
   <p><%= t(".accessibility_labels_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_accessibility_labels" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_accessibility_labels", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="adjacent-links" class="design__section">
@@ -71,7 +71,7 @@
 
   <p><%= t(".adjacent_links_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_adjacent_links_for_the_same_resource" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_adjacent_links_for_the_same_resource", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="dynamic-changes" class="design__section">
@@ -79,7 +79,7 @@
 
   <p><%= t(".dynamic_changes_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_dynamic_functionality_changes_the_page_context_unintuitively" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_dynamic_functionality_changes_the_page_context_unintuitively", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="aria" class="design__section">
@@ -87,7 +87,7 @@
 
   <p><%= t(".use_aria_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_use_aria_attributes_where_possible" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_use_aria_attributes_where_possible", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
 
 <section id="elements-hidden" class="design__section">
@@ -95,5 +95,5 @@
 
   <p><%= t(".elements_hidden_paragraph") %></p>
 
-  <p><%= t(".see_further_info") %> <a href="https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_elements_hidden_from_the_accessibility_api" target="_blank" rel="noopener noreferrer"><%= t(".here_link") %></a></p>
+  <p><%= t(".see_further_info_html", link: link_to(t(".here_link"), "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_elements_hidden_from_the_accessibility_api", target: "_blank", rel: "noopener noreferrer")) %></p>
 </section>
diff --git a/decidim-design/config/locales/en.yml b/decidim-design/config/locales/en.yml
index 69c2237be8..c479dbdb91 100644
--- a/decidim-design/config/locales/en.yml
+++ b/decidim-design/config/locales/en.yml
@@ -21,12 +21,12 @@ en:
           heading_on: Heading on important sections
           here_link: here
           illogical_heading_order: Illogical heading order
-          illogical_heading_paragraph: Every page should have a logical heading order when using the &#60;h1&#62;, &#60;h2&#62;, &#60;h3&#62;, &#60;h4&#62;, &#60;h5&#62; and &#60;h6&#62; heading elements.
+          illogical_heading_paragraph: "Every page should have a logical heading order when using the <h1>, <h2>, <h3>, <h4>, <h5> and <h6> heading elements."
           important_sections_heading: Heading on important sections
           important_sections_paragraph: It is highly important that each important section of the page has a heading to make it easier to understand what important sections are on the page just by browsing through its headings.
           links_and_buttons: Links and Buttons
           links_and_buttons_paragraph: The anchor elements (i.e. links) are meant to link to different pages or to anchor positions within the page. If the element is supposed to e.g. open some hidden item on that page, you should use the &#60;button&#62; element instead.
-          see_further_info: See further info
+          see_further_info_html: "See further info %{link}"
           unique_h1: Unique H1
           unique_h1_paragraph: Every page should have a unique H1 heading on it
           use_aria: Use ARIA

```

Reply:
Hi, in my local development environment, writing `<p><%= t(".see_further_info_html", url: "https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_illogical_heading_order", target: "_blank", rel: "noopener noreferrer") %></p>` in accessibility.html.erb and `see_further_info_html: See further info <a href="%{url}" target="%{target}" rel="%{rel}">here</a>` in en.yml worked as expected.

I think it's better to avoid using nested `t()` functions as much as possible.

Reply:
yes, that is also a way to do it :) 

Also, you could assume that all the links have the same 2 classes and same target, and you could remove from injecting them.

Reply:
Thanks. This page now been updated. Will request to merge shortly @alecslupu.